plugin.yaml

```diff
@@ -7,6 +7,8 @@ configuration:
   properties:
     name:
       type: string
-    policy:
+    permissionspolicy:
```

Comment:
Can we kebab-case the options?

Reply:
does that work ?

Reply:
- https://github.com/buildkite-plugins/ecr-buildkite-plugin/blob/df34ad30e592c44e450f02e14366f75bf9beb237/plugin.yml#L17
- https://github.com/buildkite-plugins/ecr-buildkite-plugin/blob/df34ad30e592c44e450f02e14366f75bf9beb237/hooks/pre-command#L69